README.md

```diff
@@ -38,7 +38,7 @@ import useDropdownMenu from 'react-accessible-dropdown-menu-hook';
 Call the Hook, telling it how many items your menu will have:
 
 ```tsx
-const { buttonProps, itemProps, isOpen } = useDropdownMenu(numberOfItems);
+const { buttonProps, itemProps, isOpen } = useDropdownMenu<T>(numberOfItems);
```

Comment:
Can you update this to be more explicit about what `T` is? Probably to something like `HTMLElement`.